nmostesting/TestHelper.py

```diff
@@ -264,6 +268,48 @@ def loader(uri):
     return schema
 
 
+def read_RSA_private_key(priavte_key_files):
+    for private_key_file in priavte_key_files:
+        private_key = open(private_key_file, "r").read()
+        if private_key.find("BEGIN RSA PRIVATE KEY") != -1:
+            return private_key
+    return None
+
+
+def generate_jwk(rsa_private_key):
+    rsa_key = RSA.importKey(rsa_private_key)
+    public_key = rsa_key.publickey().exportKey(format="PEM")
+    return JsonWebKey.import_key(public_key, {"kty": "RSA", "use": "sig",
+                                 "key_ops": "verify", "alg": "RS512"}).as_dict()
+
+
+def generate_token(rsa_private_key, scopes=None, write=False, azp=False, add_claims=True, overrides=None):
+    if scopes is None:
+        scopes = []
+    header = {"typ": "JWT", "alg": "RS512"}
+    payload = {"iss": "{}".format("https://testsuite.nmos.tv"),

```

Comment:
This can be fixed, not based on CONFIG.DNS_DOMAIN, right?

Reply:
Thanks, I am looking into it.

Reply:
Fixed